requirements2.txt

```diff
@@ -137,6 +137,7 @@ nibabel==5.2.0
 ninja==1.11.1.1
 numba==0.60.0
 numcodecs==0.12.1
+numpy==1.25.2
```

Comment:
is numpy in both requirements? is this correct?

Reply:
I did it on purpose because numpy is required at build time and at runtime. If we were writing the requirements files by hand I wouldn't have included it in both, but I am trying to create some logic to the separation of the two requirements files and the relationship to the pyproject.toml.